api/src/models/project-view.ts

```diff
@@ -5,6 +5,18 @@ import { getLogger } from '../utils/logger';
 
 const defaultLog = getLogger('models/project-view');
 
+export interface IGetProject {
+  id: number;
+  coordinator: GetCoordinatorData | null;
+  permit: any;
+  project: any;
+  objectives: GetObjectivesData | null;
+  location: GetLocationData | null;
+  iucn: GetIUCNClassificationData | null;
+  funding: GetFundingData | null;
+  partnerships: GetPartnershipsData | null;
```

Comment:
small nits: 
There are interfaces defined for permit and project that we can assign here.
Also, are these ever null? Its only used in 1 function and I think it always returns an object for each of these?

Reply:
So I added types for Permit and Project. Nice catch! 
About the nulls, we need them as they are used in `getProjectEntitiesById` which only return entities that are requested and else others will be null.